### ISSUE TITLE
Speecbubble game crash

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#241]
```
What steps will reproduce the problem?
1. Type "4 something" in a game with only 3 hogs per team
2. Type "4 something" in a game where certain hogs died
3. Use - and' and ` instead of " with same effects

What is the expected output? What do you see instead?
A speechbubble for the 4th hog is expected. Instead everyone elses game crashes 
and your game looks like the others lag. But you can still read everyone's 
comments.

What version of the product are you using? On what operating system?
HW 0.9.15 on Ubuntu 11.10

Please provide any additional information below.


```

-----
Original issue reported on code.google.com by `Johanna...@gmail.com` on 8 Jul 2011 at 2:40
* Merged into: #137

**Comment by GoogleCodeExporter** [comment]
```
nemo  is fully aware of this and this bug was fixed a day after releaseing 
9.15. 
```

-----
Original comment by `davidb...@gmail.com` on 9 Jul 2011 at 8:50

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Yes like I said it was fixed AFTER 9.15 and will not be there in .16
```

-----
Original comment by `davidb...@gmail.com` on 9 Jul 2011 at 10:19

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `kyberneticist@gmail.com` on 9 Jul 2011 at 10:51
* Changed state: **Duplicate**
